app/controllers/users.js

```diff
@@ -41,3 +40,31 @@ exports.authenticate = (req, res, next) => {
     })
     .catch(reason => next(errors.defaultError(`Database error - ${reason}`)));
 };
+const hasSomething = param => param !== null && param !== '';
+exports.list = (req, res, next) => {
+  jwt.verify(req.headers.session_token, JWT_KEY, function(jwtError, decoded) {
```

Comment:
promisify this method and move it outside the controller

Reply:
👍